recipes/poseidon-xerxes/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1.0.0" %}
+package:
+  name: poseidon-xerxes
+  version: {{ version }}
+
+source:
+  - url: https://github.com/poseidon-framework/poseidon-analysis-hs/releases/download/v{{ version }}/xerxes-linux # [linux]
+    md5: 7b91c8fbc22e8aa3315a7f17af78aafe # [linux]
+  - url: https://github.com/poseidon-framework/poseidon-analysis-hs/releases/download/v{{ version }}/xerxes-macOS # [osx]
+    md5: 6896e59a0023bf8c882b6cd775dff059 # [osx]
+
+build:
+  number: 0
+  script:
+    - "mkdir -p $PREFIX/bin"
+    - "mv xerxes* $PREFIX/bin/xerxes"
+    - "chmod u+x $PREFIX/bin/xerxes"
+
+requirements:
+  build:
+    # Precompiled binaries still link against libgcc
+    - {{ compiler('c') }}
```

Comment:
Is there any way we could compile these? Precompiled binaries tend to break in our experience.

Reply:
I have tried very hard. Definitely not for Mac, but potentially at some point for Linux. The issue is that the CentOS7 system that bioconda builds on is poorly supported by Haskell build tools, so there are _a lot_ of hoops to jump through to make it work, and I did not succeed. 

Note that I already did go to quite some lengths to get a CentOS7 version pre-compiled through a Docker image. It's the only combination that so far worked for me to get Haskell executables on to bioconda (see also https://github.com/bioconda/bioconda-recipes/tree/master/recipes/poseidon-trident, where we've used the same strategy after several weeks of trying direct compilations)